src/home/HomePage.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class='homepage'>
+    <tasks-list :taskList='taskList' v-on:complete='onComplete($event)' />
```

Comment:
follow camel case for components eg: `TasksList`

Reply:
ok, will do